pmd/dependencies.bzl

```diff
@@ -15,8 +15,8 @@ def rules_pmd_dependencies():
 
     # Java
 
-    rules_java_version = "4.0.0"
-    rules_java_sha = "34b41ec683e67253043ab1a3d1e8b7c61e4e8edefbcad485381328c934d072fe"
+    rules_java_version = "5.0.0"
+    rules_java_sha = "ddc9e11f4836265fea905d2845ac1d04ebad12a255f791ef7fd648d1d2215a5b"
```

Comment:
```
WARNING: Download from https://github.com/bazelbuild/rules_java/releases/download/5.0.0/rules_java-5.0.0.tar.gz failed: class java.io.FileNotFoundException GET returned 404 Not Found
```
Seems like there is no manual `tar.gz` upload. The automatic GitHub one can be used instead.
```
https://github.com/bazelbuild/rules_java/releases/download/{v}/rules_java-{v}.tar.gz
```
⬇️ 
```
https://github.com/bazelbuild/rules_java/archive/refs/tags/{v}.tar.gz
```